### PR TITLE
Loads

### PR DESCRIPTION
**Pull request by ImMPrada**
Closes https://github.com/PradaIng/structuraid-core/issues/4

https://www.notion.so/Developing-Notes-fd9915fbdcdd45b788b6801f65775a31?p=3c7b3124a1d3443f8a2d0abdc63e005f&pm=c

### CODE REVIEW

File: lib/engineering/base.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module Engineering
```
Comment (mapra99) [inline lib/engineering/base.rb:1]: what's the purpose of this namespace? It looks to me like the name might be too open for many things

**Review by mapra99 — APPROVED** [review]
it's looking nice! 🤩

File: lib/engineering/base.rb (hunk 1 above)
Comment (ImMPrada) [reply lib/engineering/base.rb:1]: Yeah, I was thinking of using it for all engineering maths that could not be in Math ruby's namespace

Comment (mapra99) [reply lib/engineering/base.rb:1]: ok, let's see how it matures

**Pull request merged**